stored_procedures/get_details_for_destination_plate.sql

```diff
@@ -0,0 +1,19 @@
+-- drop the stored procedure
+DROP PROCEDURE IF EXISTS `biosero_uat`.`getDetailsForDestinationPlate`;
+
+-- create the stored procedure
+DELIMITER $$
+
+-- Fetches the details for the specified destination plate barcode using the view
+CREATE PROCEDURE `biosero_uat`.`getDetailsForDestinationPlate` (
+  IN input_source_barcode VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci
```

Comment:
Should this be named `input_destination_barcode`

Reply:
Yes, changed